tasks/page/makeReadme.js

```diff
@@ -10,40 +10,49 @@ const converter = new htmlToJsx({createClass: true, outputClassName: 'Readme'})
 
 if (require.main !== module) {
   module.exports = function(config, folder) {
+    const folders = folder.split('/')
     const pattern = `${folder}/readme?(\.markdown|\.mdown|\.mkdn|\.md|\.rst)`
     const readmes = glob.sync(pattern, {nocase: true})
     const deps = [`build/.temp/${folder}/info.json`]
     if (readmes.length > 0) {
       deps.push(readmes[0])
     }
+
     const targets = [`build/.temp/${folder}/readme.jsx`]
+
+    if (folders.length > 4) {
+      const projectPath = folders.splice(4).join('/')
+      targets.push(projectPath)
+    }
+
     return {deps, targets, moduleDep: false}
   }
 } else {
   const {deps, targets} = utils.processArgs(process.argv)
   const readmeJsx = targets[0]
+  const multiProjectPath = targets[1]
   let html = ''
   const info = require(__dirname + '/../../' + deps[0])
   const readme = deps[1]
   if (readme != null) {
     const pkg = {repository: {url: info.repo}}
     let contents = fs.readFileSync(readme, 'utf8')
-    // replace blob image urls with raw image urls, they don't work outside of github
-    contents = contents.replace(
-      RegExp(
-        '(' +
-          escapeRegExp('https://github.com/') +
-          '.*?' +
-          '/)(blob)(/master/' +
-          '.*?.(:?png|jpeg|jpg|gif|bmp))',
-        'gi'
-      ),
-      '$1raw$3'
-    )
-    let markdown = addSpacing(contents)
+    let markdown = ''
+
     if (path.extname(readme).toLowerCase() === '.rst') {
       markdown = rst2mdown(contents)
+    } else {
+      markdown = contents
     }
+
+    if (info.repo.includes('github')) {
```

Comment:
This would match e.g. `https://bitbucket.com/foo/my-github-import`

Reply:
Good point; that wasn't the best choice.

I will change it to `https://github.com`